### DISCUSSION TITLE
mqtt integration

### DISCUSSION THREAD

**Discussion by jurajs5** [marcbelmont/deep-license-plate-recognition#101] [Ideas]
hi, is it possible to add mqtt integration? will help a lot.

**Comment by marcbelmont** [comment]
What do you have in mind? How are you trying to integrate our product?

**Comment by jurajs5** [comment]
Smart home intergration. in this case based on mqtt information open gate if my car is detected.

**Comment by marcbelmont** [comment]
Which smart home integration are you referring to?
Are you able to list the steps needed to go from reading the license plate
to opening the gate?
